### ISSUE TITLE
Compare and contrast this package with expo-crypto in the README

### ISSUE THREAD

**Issue by pcowgill** [tradle/react-native-crypto#42]
It would be helpful to compare and contrast this package with `expo-crypto` in the README

https://docs.expo.io/versions/latest/sdk/crypto/
https://www.npmjs.com/package/expo-crypto

**Comment by mvayngrib** [comment]
@pcowgill i agree, but i won't have time for this...i might accept a PR though :)

**Comment by mvayngrib** [comment]
@pcowgill for clarify, the main purpose of all these packages (rn-nodeify, react-native-randombytes, react-native-crypto) is compatibility with the node.js apis. `expo-[crypto-something]` doesn't have this goal

**Comment by giacomocerquone** [comment]
@mvayngrib mh actually it has this exact goal (among providing a way to let devs encrypt stuff of course) https://docs.expo.io/versions/latest/sdk/crypto/, the first line:

"expo-crypto enables you to hash (encrypt) data in an equivalent manner to the Node.js core crypto API"

**Comment by mvayngrib** [comment]
@giacomocerquone expo-crypto may provide the same functionality, but it provides a different interface than node.js. `react-native-crypto` lets you `require('crypto')` and reuse your node.js code

**Comment by giacomocerquone** [comment]
Well i'd do a pr to explain this as a starting point to "showcase" the differences
